modules/nf-core/blast/updateblastdb/main.nf

```diff
@@ -0,0 +1,50 @@
+process BLAST_UPDATEBLASTDB {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/blast:2.15.0--pl5321h6f7f691_1':
+        'biocontainers/blast:2.15.0--pl5321h6f7f691_1' }"
+
+    input:
+    tuple val(meta)
+
+    output:
+    tuple val(meta), path("${meta.id}"), emit: db
+    path "versions.yml"                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
```

Comment:
Align equal signs

Reply:
Hi, do you mean the '=' of args and prefix vertically? If that, other modules don't look they are doing it.